docs/testing/testplan.md

```diff
@@ -0,0 +1,344 @@
+| R4   | Lack of understanding about the standard or the technology | high     | <ul><li>Provide accessible, good quality resources to simplify understanding:<ul><li>Web Monetization</li><li>The extension</li><li>How to start using the extension</li></ul></li><li>Provide good quality of docs (accurate, comprehensive, complete)</li><ul><li>How Web Monetization works & what to expect</li><li>Understanding the technology</li><li>Access to the [proposed Web Monetization standard specification](https://webmonetization.org/specification/) & the working group [WICG](https://github.com/WICG/webmonetization) GitHub repository</li></ul></ul> |
+| R5   | UX that is poor                                            | critical | <ul><li>Provide user experience (UX) that supports ease of use</li><li>Provide fast and responsive help desk support to resolve issues</li><li>Gather user feedback, and continuously improve UX and features</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                  |
+| R6   | Unable to control payments                                 | critical | <ul><li>Ability to control when, who & how much to pay</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+<!-- prettier-ignore-end -->
+
+#### Basic Test Cases
+
+##### Connect to a wallet
+
+**Test ID**: 1  
+**Description**: Connect the extension to a Web Monetization enabled digital wallet
+**Risk**: R2 (onboarding) and R5 (UX)  
+**Preconditions**:
+
+1. You have already downloaded and installed the Web Monetization extension.
+2. Your digital wallet sign-up and account activation is complete (learn more from the [Prerequisites](#Prerequisites) section)
+
+**Steps**:
+
+1. Open the WM extension and copy the extension’s key.
+2. Open the digital wallet, and load the extension’s key into the wallet.
```

Comment:
This is soon going away. Might want to wait for https://github.com/interledger/web-monetization-extension/issues/613

Reply:
Instead of holding back this PR, let me suggest that we earmark an update to the test plan when the tasks for #613 are merged.

Reply:
#613 is closed. Should this be updated?